src/dmd/dinterpret.d

```diff
@@ -4905,9 +4938,15 @@ public:
         }
 
         if (isTrueBool(econd))
+        {
             result = interpret(pue, e.e1, istate, goal);
+            incUsageCtfe(istate, e.e1.loc);
```

Comment:
doesn't this wind up counting the line for e1 twice?

Reply:
It does not.
We only increment on statements.
e1 is not a statement.